Week3/actions.js

```diff
@@ -0,0 +1,152 @@
+const util = require("util");
+const mysql = require("mysql");
+const test = require("./importData.js");
+
+// 1- create connection
+const connection = mysql.createConnection({
+  host: "localhost",
+  user: "root",
+  password: "mosleh1234"
+});
+// 2- connect
+
+connection.connect(err => {
+  if (err) {
+    throw err;
+  }
+  console.log("Mysql connected");
+});
+// // 3- Create database
+let sql = "CREATE DATABASE IF NOT EXISTS todo_app";
+connection.query(sql, err => {
+  if (err) throw err;
+});
+// 4- use database my_todo
+connection.query("USE todo_app", err => {
+  if (err) throw err;
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function showTable(req, res) {
+  const tableName = req.params.table;
+  try {
+    const showRecords = await execQuery(`SELECT * FROM ` + tableName);
+    res.status(200).send(showRecords);
+  } catch (error) {
+    return res.status(404).send("error");
+  }
+  res.end();
+}
+async function createUser(req, res) {
+  try {
+    await execQuery(`INSERT INTO users (user_name,email) VALUES (?,?)`, [
+      req.body.user_name,
+      req.body.user_name
+    ]);
+    res.status(201).send("new user inserted");
```

Comment:
I think it could be great to return the created user, so I could, for instance, read the created id and using it for another call.